src/SDKs/EventGrid/EventGrid.Tests/EventGrid.Tests.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>EventGrid.Tests</PackageId>
+    <VersionPrefix>1.0.0-preview</VersionPrefix>
+    <Description>EventGrid.Tests Class Library</Description>
+    <Authors>Microsoft Corporation</Authors>
+    <AssemblyName>EventGrid.Tests</AssemblyName>
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Management.EventGrid\Microsoft.Azure.Management.EventGrid.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Update="SessionRecords\**\*.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+
+  <!--Do not remove until VS Test Tools fixes #472-->
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
+  </ItemGroup>
+  <ItemGroup>
+    <Folder Include="SessionRecords\EventGrid.Tests.ScenarioTests.ScenarioTests\" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Update="xunit" Version="2.3.0-beta4-build3742" />
```

Comment:
@kalyanaj all these references are already added, these are redundant.
We would like to reference certain common nuget references from a common location which helps in upgrading.